features/login.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Logging in, Logging out
+
+  @javascript
+  Scenario: I can log-in and out
+    Given I am logged in as an admin
+    And I am on the Dashboard
+    Then I should see "Howdy"
+
+    Given I am an anonymous user
```

Comment:
It's meant to test the `Given I am an anonymous user`  step to ensure it actually logs the user out. (Additionally that `Given I am logged in as...` step actually logs you in). If I were testing an application then it should be `When I log out` (if I were testing logging out), but here we're really testing WordHat's step implementations.